docs/AppendixA-FAQ.md

```diff
@@ -1 +1,69 @@
 # Appendix A - FAQ
+
+### A.1 Installation trouble shooting notes
+
+Key configuration points to verify:
+
+- Depending on your operating system, some dependencies might be missing.
+    During initial troubleshooting of the imaging pipeline, note any related
+    error messages (e.g. `install_driver(mysql) failed: Can't locate DBD/mysql
+   .pm`) and install missing packages as needed (e.g.
+   `sudo apt-get install libdbi-perl`, `sudo apt-get install libdbd-mysql-perl`,
+    `sudo apt-get install libarchive-zip-perl`).
+
+- If your MINC toolkit is older than 1.9.14 and your scans have no Date of Birth
+   value, you may see an age unit error during DICOM to MINC conversion.
+   Instructions for compiling a more recent version available on
```

Comment:
"...more recent version **of the MINC toolkit are** available on..."

Reply:
ok